src/Campaigns/Models/Campaign.php

```diff
@@ -110,6 +110,16 @@ public static function find($id): ?Campaign
         return give(CampaignRepository::class)->getById($id);
     }
 
+    /**
+     * Find campaign by Form ID
+     *
+     * @unreleased
+     */
+    public static function findByFormId($id): ?Campaign
```

Comment:
Same thing here, I added a method with the same name on #7571 so we'll need to manage this conflict later.